presidio_evaluator/tag_to_span.py

```diff
@@ -0,0 +1,62 @@
+from typing import List, Optional
+from spacy.tokens import Doc
+
+from presidio_evaluator import Span
+
+
+def _to_io(tag):
+    if "-" in tag:
+        return tag[2:]
+    return tag
+
+
+def tag_to_span(tokens: Doc, tags: List[str]) -> List[Span]:
+    """
+    Transforms a list of tags to a list of spans, while ignoring labeling scheme.
+    :param tokens: List of spaCy tokens
+    :param tags: List of tags (e.g. ["O", "B-PER", "I-PER", "U-LOC"])
+    :return: List of spans
+    """
+
+    # If IO, translate to BIO for easier manipulation
```

Comment:
should it be "if it is not IO, translate to IO for easier manipulation"?

Reply:
Yes! good catch